docs/git-experts.md

```diff
@@ -0,0 +1,74 @@
+# Jujutsu for Git experts
+
+People who are proficient with Git often ask what benefit there is to using
+Jujutsu.
+
+## Git is still available to use
+
+Jujutsu repositories are colocated by default, which means they can be used by
+both the `jj` and `git` commands.
+
+If you encounter something you can't do with Jujutsu or is simply easier with
+Git, you have the option of using `git` for that command, then switching back.
+
+## Automatic rebasing makes for less toil
+
+> I already know how to use `git rebase --interactive`.
+
+Rebasing is automatic, so it's one fewer step to take. As soon as you've am.
+
+Git:
+
+- stage
+- commit
+- rebase
+
+Jujutsu:
+
+- squash -i --into
+
+TODO: Show example
+
+TODO: Acknowledge `git rebase --update-refs`
+
+## Undo is easier than searching the reflog
```

Comment:
I would say "is more powerful than the reflog" to focus more appropriately, it's about ease of use *and* power, but you want to focus on power first for a git audience.

Reply:
BTW, if I *can* do this, with jj's rich output, timestamps, summary of what operation triggered the change, etc., I would be thrilled to learn this! But the current text doesn't say how.

Reply:
Correct, we don't have a simple way of walking the operation log filtered to a specific ref yet. #2871 is the FR specifically for doing that for the working copy.